data/ecosystems/m/multiversx-elrond.toml

```diff
@@ -2001,3 +2001,198 @@ url = "https://github.com/zdenal/elrond-amm"
+url = "https://github.com/CheckerChain"
+
+[[repo]]
+url = "https://github.com/Chessout/chessout"
+
+[[repo]]
+url = "https://github.com/CoinDrip-finance/protocol-v2"
+
+[[repo]]
+url = "https://github.com/CosmicF?tab=repositories"
+
+[[repo]]
+url = "https://github.com/newtmex/CryptGain-submission-for-xDay-hack-2023"
+
+[[repo]]
+url = "https://github.com/radupara/decentralized-copyright-shield"
+
+[[repo]]
+url = "https://github.com/thedefibuilder/xdays-xdefigpt"
+
+[[repo]]
+url = "https://github.com/Entangle-Protocol/"
+
+[[repo]]
+url = "https://github.com/doinel1a/gen-x"
+
+[[repo]]
+url = "https://github.com/E-Compass/hackathon_portfolio_ui/"
+
+[[repo]]
+url = "https://github.com/GaupaLabs/Gaupa-Tracker-App"
+
+[[repo]]
+url = "https://github.com/GiveCup"
+
+[[repo]]
+url = "https://github.com/butavpaul/GOFClient"
+
+[[repo]]
+url = "https://github.com/Inspir3-io/Inspir3"
+
+[[repo]]
+url = "https://github.com/defralcoding/InvoiceXpress"
+
+[[repo]]
+url = "https://github.com/JewelSwapXFinance/jewel-flashmint-sc"
+
+[[repo]]
+url = "https://github.com/orgs/locki-io/repositories"
+
+[[repo]]
+url = "https://github.com/esdt-space/lunar-pay-web"
+
+[[repo]]
+url = "https://github.com/buidly/sc-lythra-rs"
+
+[[repo]]
+url = "https://github.com/buidly/sc-materia-prima"
+
+[[repo]]
+url = "https://github.com/EmanHerawy/multi-vest-dao"
+
+[[repo]]
+url = "https://github.com/markeljan/multiversx-ai-dev"
+
+[[repo]]
+url = "https://github.com/Blockchain-Hackers/multiversx-toolchain"
+
+[[repo]]
+url = "https://github.com/multihx/multiversx-orderbook.git"
+
+[[repo]]
+url = "https://github.com/Mer0ps/mvx-snap"
+
+[[repo]]
+url = "https://github.com/CommanderAstern/encode-multiversX"
+
+[[repo]]
+url = "https://github.com/gfusee/novax"
+
+[[repo]]
+url = "https://github.com/aperson101/nwallet"
+
+[[repo]]
+url = "https://github.com/oort-digital/Oort-Loyalty-and-Rewards-Whitelist-Tools"
+
+[[repo]]
+url = "https://github.com/DamDev1/Orbix"
+
+[[repo]]
+url = "https://github.com/AIRO-NETWORK/pdfnft-backend"
+
+[[repo]]
+url = "https://github.com/Blockchain-Hackers/prime-wallet"
+
+[[repo]]
+url = "https://github.com/KhatuntsevAlex/Hakaton_swap_pro_mode/tree/main"
+
+[[repo]]
+url = "https://github.com/astrarizon/hackathon-devtool/"
+
+[[repo]]
+url = "https://github.com/PulsarMoney/hackathon-pulsar-v2"
+
+[[repo]]
+url = "https://github.com/TudorEsan/pulsar-wallet"
+
+[[repo]]
+url = "https://github.com/alexbolog/refracto-sc-crowdfund"
+
+[[repo]]
+url = "https://github.com/vampirebyte/remotis-mx-contracts-rs"
+
+[[repo]]
+url = "https://github.com/Pruvendo/multiversx-hackathon-artifacts"
+
+[[repo]]
+url = "https://github.com/autoscale-defi/savings-rs"
+
+[[repo]]
+url = "https://github.com/en0c-026/scenario-mx-studio"
+
+[[repo]]
+url = "https://github.com/SenDevSei/Senseidevtest"
+
+[[repo]]
+url = "https://github.com/michavie/mx-hackathon"
+
+[[repo]]
+url = "https://github.com/Azerus972/tikaw"
+
+[[repo]]
+url = "https://github.com/MVX-TKYC"
```

Comment:
This is an org link, not a repo link. Please add this org link to the org list section at the top of the file if you consider that the entire org is building / built in MultiversX.